### DISCUSSION TITLE
Not receiving notifications

### DISCUSSION THREAD

**Discussion by diip3sh** [community/community#132542] [Mobile]
I have been using github mobile for a while now, previously I was able to receive notification but recently after updates I'm unable to get any notification from the app. I have to look into manually in notifications section to see it.

For more references, this is my settings
![image](https://github.com/user-attachments/assets/e9f79b19-26e6-493c-bed5-65a7b6345255)
<details>
Phone notifications setting
![image](https://github.com/user-attachments/assets/d9b3183c-8cd2-4f9c-abbb-0b553a633912)
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.169.0 (136088047)
Device: iPhone 13 (iOS 18.0)
</details>

**Comment by danialtavakoli** [comment]
If you're facing issues with GitHub Mobile not delivering notifications after recent updates, there are a few steps you can take to troubleshoot and potentially resolve the issue:

### 1. **Check Notification Settings in the GitHub Mobile App**

- Open the GitHub Mobile app on your device.
- Go to the **Settings** section of the app (usually accessible from your profile icon or the menu).
- Look for **Notification Settings** or a similar option.
- Ensure that notifications are enabled for the types of activities you want to receive notifications for (e.g., Pull Requests, Issues, Discussions).

### 2. **Check Device Notification Settings**

Sometimes, app notifications can be affected by your device's settings. Here’s how you can ensure notifications are enabled:

- **Android:**
  - Go to **Settings** on your device.
  - Tap on **Apps & notifications**.
  - Find and tap on **GitHub**.
  - Ensure that notifications are enabled for GitHub.

- **iOS:**
  - Go to **Settings** on your device.
  - Scroll down and tap on **GitHub**.
  - Make sure that **Allow Notifications** is turned on.

### 3. **Check Battery Optimization Settings**

On some devices, aggressive battery optimization settings can prevent apps from sending notifications in a timely manner. To adjust this:

- **Android:**
  - Go to **Settings** > **Battery**.
  - Look for options related to **Battery Optimization** or **Battery Saver**.
  - Make sure that GitHub is not optimized to save battery, or add it to the exceptions list.

- **iOS:**
  - Go to **Settings** > **Battery**.
  - Scroll down to **Battery Usage By App**.
  - Ensure that GitHub is allowed to refresh in the background.

### 4. **Update GitHub Mobile App**

Ensure that you are using the latest version of the GitHub Mobile app from the Google Play Store (Android) or the App Store (iOS). Developers often release updates to fix bugs and improve app performance, including notification issues.

### 5. **Restart Your Device**

Sometimes, simply restarting your device can resolve temporary glitches or issues with app notifications.

### 6. **Check GitHub Status**

Occasionally, issues with notifications might be due to server-side problems or maintenance on GitHub's end. You can check GitHub’s status page (status.github.com) to see if there are any ongoing issues affecting notifications.

### 7. **Contact GitHub Support**

If none of the above steps resolve your issue, it might be beneficial to reach out to GitHub Support. They can provide more specific assistance and investigate if there are any known issues with the app that might be causing notification problems.

By following these steps, you should be able to diagnose and hopefully resolve the issue of not receiving notifications on GitHub Mobile after recent updates.
